wiki/Gameplay/Game_modifier/Auto/de.md

```diff
@@ -0,0 +1,62 @@
+---
+stub: true
+tags:
+  - auto
+  - autoplay
+  - auto play
+  - AT
+  - AO
+  - mod
+  - game modifier
+  - Spielmodifikation
+---
+
+# Auto (Mod)
+
+![AT Modsymbol](/wiki/shared/mods/AT.png "Auto (AT) Modsymbol")
+
+*Für die vollständige Liste aller Mods, siehe: [Spielmodifikationen](/wiki/Game_modifier)*\
+*Nicht zu verwechseln mit [Autopilot (Mod)](/wiki/Game_modifier/Autopilot).*
+
+## Übersicht
+
+- Abkürzung: AT oder AO
+- Typ: Sonstige
+- Score-Multiplikator: 1,00x
+- Standard-Tastenkürzel: `V`
+- Untertitel: `Sieh dir einen perfekten Durchlauf durch das Lied an.`
+- Kompatible Spielmodi: ![][osu!] ![][osu!taiko] ![][osu!catch] ![][osu!mania]
+
+## Beschreibung
+
+*Hinweis: Das Aktivieren der Mod Auto sorgt dafür, dass das Spiel nicht gerankt wird und nicht in der lokalen Rangliste gespeichert wird.*
+
+Die Mod **Auto** (auch *Autoplay* genannt) ist eine [Spielmodifikation](/wiki/Game_modifier), die es Spielern ermöglicht, einen perfekten Durchlauf der ausgewählten Beatmap zu sehen.
+
+Die Mod funktioniert genau wie ein [Replay](/wiki/Gameplay/Replay), bei dem die Optionen für die 0,5-fache/1-fache/2-fache Wiedergabegeschwindigkeit zur Verfügung stehen und Kommentare auf der Beatmap eingesehen sowie erstellt werden können. Bis auf ein paar Ausnahmen wird immer die Note SS erlangt.
+
+### osu!
+
+In [osu!](/wiki/Game_mode/osu!) wird die Mod Auto beim Erscheinen eines Spinners den Spinner bei einer maximalen Umdrehung von 477 Spins pro Minute (SPM) gegen den Uhrzeigersinn drehen.
+
+### osu!taiko
+
+In [osu!taiko](/wiki/Game_mode/osu!taiko) wird die Mod Auto versuchen, einen perfekten Durchlauf zu erzielen, aber erlangt nicht immer die Note SS. Sie wird gelegentlich einen Tick bei Drumrolls, die nicht auf einem ganzen oder halben Takt starten, verfehlen und wird nicht immer in der Lage dazu sein, Dendens auf Beatmaps mit einer sehr hohen allgemeinen Schwierigkeit zu vervollständigen.
+
+### osu!catch
+
+In [osu!catch](/wiki/Game_mode/osu!catch) geht die Mod Auto über perfekt hinaus und nutzt unbegrenzte Geschwindigkeit, um *jede einzelne Banane* in einem Spinner zu fangen, obwohl eine solche Leistung für einen menschlichen Spieler unmöglich ist.
+
+### osu!mania
+
+In [osu!mania](/wiki/Game_mode/osu!mania) wird die Mod Auto versuchen, einen perfekten Durchlauf zu erzielen, aber verfehlt möglicherweise dicht aneinanderliegende Noten (z. B. mehrere Tastendrücke nah beieinander auf einer Hold-Note).
+
+## Trivia
+
+- Beim Zuschauen eines Replays mit der Mod Auto ist der in der Rangliste angezeigte Nutzername "osu!".
```

Comment:
```suggestion
- Beim Anschauen eines Replays mit der Mod Auto ist der in der Rangliste angezeigte Nutzername "osu!".
```

`Zuschauen` to me means watching something live. Watching a replay live sounds weird to me. Not a huge deal though.